dependencies.gradle

```diff
@@ -41,6 +41,7 @@ ext {
             glideOkHttp3       : "com.github.bumptech.glide:okhttp3-integration:${glideVersion}",
             jodaTimeAndroid    : 'net.danlew:android.joda:2.9.9.3',
             kotlin             : "org.jetbrains.kotlin:kotlin-stdlib-jre7:${kotlinVersion}",
+            kotlinReflect      : "org.jetbrains.kotlin:kotlin-reflect:${kotlinVersion}",
```

Comment:
uhm do we really need this? If I remember correctly this is something like 20K methods, looks like a bit of an overkill for a warning, even if we have proguard

Reply:
It was mentioned in some issues in `moshi-kotlin` as being included — and I did a dependencies check, we already have it in the apk anyway. Which makes sense, since the warning is about multiple versions of it being added to the classpath. This is only to set the one version we actually want to have in the app (the warning does not go away anyway, for some reason)

Reply:
Cool